crates/re_renderer/shader/rectangle.wgsl

```diff
@@ -51,6 +50,9 @@ struct UniformBuffer {
 
     minification_filter: u32,
     magnification_filter: u32,
+
+    /// Boolean: decode 0-1 sRGB gamma to linear space before filtering?
+    decode_srgb: u32,
```

Comment:
wgsl has a (32bit large) `bool` type, why not use that?

Reply:
tried to find spec on when a bool is regarded as true, now not sure if it can be in a uniform buffer. Did you try?

Reply:
NOPE

```
[2023-05-24T15:47:35Z ERROR re_renderer::error_tracker] WGPU error tick_nr=1 description=Validation Error
    
    Caused by:
        In Device::create_shader_module
        
    Shader validation error: 
       ┌─ :86:1
       │
    86 │ var<uniform> rect_info: UniformBuffer;
       │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ naga::GlobalVariable [1]
    
    
        Global variable [1] 'rect_info' is invalid
        Alignment requirements for address space Uniform are not met by [16]
        The type is not host-shareable
    
```